packages/back-end/src/controllers/features.ts

```diff
@@ -157,6 +161,12 @@ export async function getFeaturesPublic(req: Request, res: Response) {
       "public, max-age=30, stale-while-revalidate=3600, stale-if-error=36000"
     );
 
+    const setSseHeaders = IS_CLOUD ? sseEnabled ?? false : false;
```

Comment:
Instead of this, does it make sense to make `sseEnabled` returned from `getPayloadParamsFromApiKey` default to false rather than allowing it to be undefined?

Reply:
The underlying API call also supports the legacy payload which does not have `sseEnabled` in the payload. Plus we need to force it off for local installations. Probably safer to leave this alone for now.